### ISSUE TITLE
Error: "(alias) const Container: React.FC<_ReactPixi.IContainer> import Container Type '{ children: Element; x: number; y: number; }' is not assignable to type 'IntrinsicAttributes & Partial<Omit<Container<DisplayObject>"

### ISSUE THREAD

**Issue by adnanlah** [pixijs/pixi-react#353]
### Description
I get this error when I try this example:

![image](https://user-images.githubusercontent.com/8447982/193404636-844e36af-bdad-479c-84c3-a1e24d92c2fd.png)

```ts
import { Stage, Sprite, Container } from "@inlet/react-pixi";

function MapDisplay() {
  return (
    <div>
      <Stage width={300} height={300} options={{ backgroundAlpha: 0 }}>
        <Container x={150} y={150}>
          <Sprite image="https://s3-us-west-2.amazonaws.com/s.cdpn.io/693612/IaUrttj.png" />
        </Container>
      </Stage>
    </div>
  );
}

export default MapDisplay;
```

### Steps to reproduce

1. install @inlet/react-pixi and pixi.js in a Vite react app.
2. Try a small example in a React component.

### Additional info

- `@inlet/react-pixi` version: "^6.8.0"
- `React` version: "^18.2.0"
- `ReactDOM` version: "^18.2.0"
- `PIXI` version: "^6.5.4"

**Comment by saitonakamura** [comment]
@plantain-00 @jdlee022  You can use `@saitonakamura/react-pixi@17.0.0-beta-5`, it has support for react 18 and we've being using it in production for a month

**Comment by baseten** [comment]
React 18 support has landed in the latest alpha `v7.0.0-alpha.0`. Please install from `@pixi/react-pixi`
